crates/ra_syntax/src/parsing/lexer.rs

```diff
@@ -12,6 +12,21 @@ pub struct Token {
     pub len: TextUnit,
 }
 
+macro_rules! match_literal_kind {
```

Comment:
Makes sense!

Let's make this a function instead of a macro though!

Reply:
👌